docs/pushnotificationios.md

```diff
@@ -12,97 +12,101 @@ title: '🚧 PushNotificationIOS'
   </p>
 </div>
 
-Handle push notifications for your app, including permission handling and icon badge number.
+Handle notifications for your app, including scheduling and permissions.
 
-To get up and running, [configure your notifications with Apple](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AddingCapabilities/AddingCapabilities.html#//apple_ref/doc/uid/TP40012582-CH26-SW6) and your server-side system.
+---
 
-React Native version equal or higher than 0.60.0:
+# Getting Started
```

Comment:
```suggestion
## Getting Started
```

Nit, we should not use the H1, it's reserved for page titles, and within Docusaurus those headings are non-linkable.